dev-infrastructure/templates/region.bicep

```diff
@@ -47,7 +47,7 @@ module regionalZoneDelegation '../modules/dns/zone-delegation.bicep' = {
   params: {
     childZoneName: regionalDNSSubdomain
     childZoneNameservers: regionalZone.properties.nameServers
-    parentZoneName: baseDNSZoneName
```

Comment:
hmm. this would try to delegate the regional zone into the regional zone as child zone and parent zone would be the same